visualization/app/codeCharta/ui/customConfigs/uploadCustomConfigButton/uploadCustomConfigButton.component.ts

```diff
@@ -12,15 +12,19 @@ export class UploadCustomConfigButtonComponent {
 	upload() {
 		const fileInput = createCCFileInput()
 		fileInput.addEventListener("change", async () => {
-			const customConfigsContent = await Promise.all(readFiles(fileInput.files))
-			for (const customConfigContent of customConfigsContent) {
-				try {
-					CustomConfigHelper.importCustomConfigs(customConfigContent)
-				} catch {
-					// Explicitly ignored
-				}
-			}
+			void this.uploadOnEvent(fileInput)
```

Comment:
Is there a reason why this function is not awaited? Additionally, you could remove the void keyword.

Reply:
Small note about these changes: The "async" keyword was missed and will be corrected in the following commit: https://github.com/MaibornWolff/codecharta/pull/3273/commits/6da6bdcd7e0e6fc14eb2bae45fcbe4f6c14d4b9e